sw/device/tests/sram_ctrl_execution_test_ret.c

```diff
@@ -111,7 +111,11 @@ typedef void (*sram_ctrl_instruction_fault_function_t)(void);
  * https://github.com/riscv-non-isa/riscv-asm-manual/blob/master/riscv-asm.md
  */
 static void sram_execution_test(void) {
-  memset((void *)kRetRamStartAddr, 0, SRAM_CTRL_TESTUTILS_DATA_NUM_BYTES);
+  uint8_t *ram_start_addr = (uint8_t *)kRetRamStartAddr;
```

Comment:
Why is this changed?

Reply:
This was failing due to a boundary overflow, even with the cast. I can go back a look if I was using a different primitive for memset.

Reply:
Ok, this error manifested because I built the latest RiscV toolchain.  Back revved to 9.2.0 and now this error doesn't manifest.